doc/sheet_functions.md

```diff
@@ -23,6 +23,9 @@ Each asset record includes: ticker, name, slug, price in USD, market cap in USD,
 volume in USD, USD balance, ETH balance, ETH spent in the last 30 days,
 ETH spent in the last 7 days, ETH spent in the last day.</p>
 </dd>
+<dt><a href="#SAN_BITMEX_PERPETUAL_CONTRACT_FUNDING_RATE">SAN_BITMEX_PERPETUAL_CONTRACT_FUNDING_RATE(projectSlug, from, to, interval)</a> ⇒ <code>Array</code></dt>
+<dd><p>Returns the funding rate that is paid by one side of the perpetual contract to the other.</p>
```

Comment:
Does not sound good to me, could be only me :) 
Isn't: 
`Returns the funding rate that is paid by one of the sides of the perpetual contract to the other`
better?